syncd/syncd_hard_reinit.cpp

```diff
@@ -717,15 +817,24 @@ sai_unicast_route_entry_t getRouteEntryFromString(const std::string &strRouteEnt
     return routeEntry;
 }
 
-void processRoutes()
+void processRoutes(bool defaultOnly)
 {
     SWSS_LOG_ENTER();
 
+    SWSS_LOG_TIMER("apply routes");
+
     for (auto &kv: g_routes)
     {
         const std::string &strRouteEntry = kv.first;
         const std::string &asicKey = kv.second;
 
+        bool isDefault = strRouteEntry.find("/0") != std::string::npos;
+
+        if (defaultOnly ^ isDefault)
+        {
+            continue;
+        }
+
```

Comment:
now we need to enumerate all routes in the table twice? but it looks like you have benchmarked this function, so how long does it take for this function when defaultOnly is true?